android/build.gradle

```diff
@@ -41,5 +41,5 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.facebook.android:facebook-android-sdk:12.+'
+    implementation 'com.facebook.android:facebook-android-sdk:12.3.0'
```

Comment:
Hi @Ayman-Barghout. Thank you for your solution. Right now, we use `+` in version to ignore update minor version in the future. I think we can use this solution to sync dependency https://stackoverflow.com/questions/35101206/android-studio-re-download-dependencies-and-sync-project